custom-fonts.md

```diff
@@ -1,13 +1,19 @@
 ---
 layout: page
-title: Using Custom Fonts
+title: Using Fonts
 
 permalink: /custom-fonts/
 ---
 
-You can use custom fonts in your Flutter application by including them in your
-`pubspec.yaml` file under the `fonts` heading in your `flutter` section:
+## Using fonts
 
+Using fonts in your Flutter application is done in two steps. 
+First include them in your `pubspec.yaml`, 
```

Comment:
describe what a "pubspec.yaml" file is. Not everyone will know that. :) e.g. "pubspec.yaml (application metadata file)"